### PR TITLE
Fix broken jsonapi controller

### PR DESCRIPTION
**Pull request by xavierdecoster**
cc @maartenba @skofman1

### CODE REVIEW

File: src/NuGetGallery/packages.config [hunk 1]
```diff
@@ -69,7 +69,6 @@
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.1.0" targetFramework="net452" />
   <package id="Modernizr" version="2.8.3" targetFramework="net452" />
   <package id="Moment.js" version="2.11.2" targetFramework="net452" />
-  <package id="MvcHaack.Ajax.MVC4" version="2.0.0.0" targetFramework="net452" />
```
Comment (maartenba) [inline src/NuGetGallery/packages.config]: Yay to less dependencies!

**Comment by maartenba** [comment]
If you fix the issue title to have a capital at the start to satisfy my OCD, :shipit:

**Comment by yishaigalatzer** [comment]
Overall looks good, just wondering why not use WebAPI since we have both running in the process? Is it for authorization reasons?

**Comment by xavierdecoster** [comment]
@yishaigalatzer to avoid regressions. Formatters are tuned for OData, so using WebAPI Controller here would involve more testing and delay the fix. Perhaps we can reconsider outside the scope of this fix?

**Comment by yishaigalatzer** [comment]
@xavierdecoster yes absolutely

**Pull request merged**